docs/docs/angular/schematics.mdx

```diff
@@ -48,6 +48,26 @@ ng g @datorama/akita:feature auth/auth --plain
 The akita-schematics: prefix is only needed when the default collection isn't set to akita-schematics
 :::
 
+### Generate a folder
+
+Add the `--flat false` to generate a feature inside its own folder. For example:
```

Comment:
It'll be better to add a concrete example as we have in the issue.

Reply:
agree

Reply:
done